docs/source/logging.rst

```diff
@@ -259,13 +259,19 @@ Configure console logging
 *************************
 
 Lightning logs useful information about the training process and user warnings to the console.
-You can retrieve the Lightning logger and change it to your liking. For example, increase the logging level
-to see fewer messages like so:
+You can retrieve the Lightning logger and change it to your liking. For example, adjust the logging level
+or redirect output for certain modules to log files:
 
 .. code-block:: python
 
     import logging
-    logging.getLogger("lightning").setLevel(logging.ERROR)
+
+    # configure logging at the root level of lightning
+    logging.getLogger("pytorch_lightning").setLevel(logging.ERROR)
```

Comment:
can we keep the previous root logger name? otherwise it is a breaking change

Reply:
I could add back this:
```python
# deprecated, will be removed in v1.4
_logger = python_logging.getLogger("lightning")
_logger.addHandler(python_logging.StreamHandler())
_logger.setLevel(python_logging.INFO)
_logger.propagate = False
```
and mention in docs that it is deprecated?

Then existing users could still use that, but we wouldn't propagate the messages up (otherwise duplicated). 
The motivation behind the change is simply standardization. User-side configuration for the library should be simply this:
```python
logger = logging.getLogger("name of package or submodule")
logger.xyz = ...
```
Does that make sense?

Reply:
Noticed we can't do this and keep the old name because __name__ starts with `pytorch_lightning`

so yeah, we have to break here